src/jquery.daterangepicker.js

```diff
@@ -1892,27 +1892,28 @@
 
 		function compare_month(m1,m2)
 		{
-			if(!m2 || !m1) {
+
+			if( m1 && m2 )
+			{
+				return parseInt(moment(m1).format('YYYYMM')) - parseInt(moment(m2).format('YYYYMM'));

```

Comment:
this changes the signature / idea of the function. A "compare" function typically returns either 1, -1, or 0...